test/e2e_node/jenkins/coreos-init.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
See `passwd.users` section, it contains the same basic information. Sorry for the re-ordering! I was going by the ignition spec for my order this time.